tests/tests_unit/test_data_model/test_models/test_dms/test_validators.py

```diff
@@ -94,9 +96,14 @@ def test_validation(client: NeatClient, valid_dms_yaml_with_consistency_errors:
 
     on_success.run(data_model)
 
-    assert len(on_success.issues) == 4
+    assert len(on_success.issues) == 7
 
     by_code = cast(IssueList, on_success.issues).by_code()
-    assert set(by_code.keys()) == {ViewsWithoutProperties.code, UndefinedConnectionEndNodeTypes.code}
-    assert len(by_code[ViewsWithoutProperties.code]) == 1
+    assert set(by_code.keys()) == {
+        ViewsWithoutProperties.code,
+        UndefinedConnectionEndNodeTypes.code,
+        VersionSpaceInconsistency.code,
+    }
+    assert len(by_code[ViewsWithoutProperties.code]) == 2
     assert len(by_code[UndefinedConnectionEndNodeTypes.code]) == 3
```

Comment:
This might not be worth it, but just pointing it out. This tests do not check that the issue message is correct. For example, if there is a logic error pointing to the wrong view, it will not be caught.